docs/Deploying.md

```diff
@@ -12,11 +12,28 @@ To make it available to your bash shell, add the following to your `.bash_profil
 
 Note:  If you are following the [recommended Ubuntu instructions](https://www.digitalocean.com/community/tutorials/how-to-install-apache-tomcat-8-on-ubuntu-14-04):  you should set ```export CATALINA_HOME=/opt/tomcat```
 
+## Prepare the global configuration file
+
+The portal is configured using a global configuration file, `portal.properties`.
+An example file is available in the `src/main/resources` folder.
+Use it as a template to create your own:
+
+    cd src/main/resources
+    cp portal.properties.EXAMPLE $HOME/cbioportal/portal.properties
+
+For more information about the `portal.properties` file,
+see the [reference](portal.properties-Reference.md) page.
+
 ## Add PORTAL_HOME to Tomcat
 
-The `PORTAL_HOME` environment variable needs to be available to the `cbioportal.war` file which runs within the Tomcat server. To make it available to Tomcat, edit your Tomcat startup file (typically `$CATALINA_HOME/bin/catalina.sh`) and add the following line anywhere within this file (we typically add it near the `JAVA_OPTS` statements):
+The `PORTAL_HOME` environment variable needs to be available to
+the `cbioportal.war` file which runs within the Tomcat server,
+so that it can find the configuration file. To make it available to Tomcat,
+edit your Tomcat startup file (typically `$CATALINA_HOME/bin/setenv.sh`)
+and add a line like the following, pointing it to the folder containing
+`portal.properties`:
 
-    export PORTAL_HOME= $CATALINA_HOME + "/webapps/cbioportal/WEB-INF/classes/"
+    export PORTAL_HOME=/Users/johndoe/cbioportal
```

Comment:
right ... the configuration can be read from anywhere now. it does not need to come from the baked in copy.